_posts/daiblej/2020-09-01-first-post-daiblej.md

```diff
@@ -0,0 +1,7 @@
+---
+layout:post
+author:daiblej
+title:Janet's first post
```

Comment:
You'll need a space after the `:` in all of these. Also, the title entry will need `""` around it: otherwise the `'` in `Janet's` will cause a minor explosion.

So the final should look like this:
```
---
layout: post
author: daiblej
title: "Janet's first post"
---
```

Cross-reference this with the instructions on the class website, which might help you realize if you misunderstood them or just mis-typed.

Reply:
How it looks like when there is no space after : and with ' instead of "? Can it still be posted on the website?